jest.config.json

```diff
@@ -12,5 +12,8 @@
   "transform": {
     "\\.[jt]sx?$": "babel-jest",
     "\\.mjs?$": "babel-jest"
+  },
+  "moduleNameMapper": {
+    "^client/(.*)$": "<rootDir>/client/$1"
```

Comment:
I wonder if we could consolidate all of these module alias wrappings to one place - the use of different syntax/functionalities in the mappings could prove tricky.

Reply:
This may not be worth the effort to investigate at this time.